dts/msm8974/msm8974-lge-hammerhead.dts

```diff
@@ -7,9 +7,12 @@
 
 / {
 	// This is used by the bootloader to find the correct DTB
-	qcom,msm-id = <0x7e 0x96 0x20002 0x2b>;
+	qcom,msm-id = <126 150 0x20002>;
```

Comment:
This is the part I'm mainly confused about. If I remember correctly the original bootloader on hammerhead wants to have this `msm-id` with 4 elements since the 0x2b at the end encodes the board revision. Are you saying that it's not actually necessary to boot lk2nd?

Reply:
"To boot lk2nd from stock bootloader", no.
The original bootloader also boots `qcom,msm-id = <126 150 0x20002>;`/`qcom,msm-id = <0x7e 0x96 0x20002>;`.

However, "to boot lk2nd from lk2nd", it's currently not possible yet, since it gets confused with the dtbs.
#285 

"To boot downstream boot/recovery image from lk2nd", aa6713e5ca1e628c13181ff89eecbc30a0313e7c is still required. And proper msm-id/board-id are also required to find proper dtbs.